apps/charterafrica/src/payload/utils/getPageUrlUsingPayload.js

```diff
@@ -0,0 +1,21 @@
+const getPageUrlUsingPayload = async (payload, slug) => {
+  const collection = "pages";
+  const options = {
+    where: {
+      slug: {
+        equals: slug,
+      },
+    },
+  };
+
+  const { docs } = await payload.find({
+    limit: 0,
+    ...options,
+    collection,
+  });
```

Comment:
Why not simplify all of this?

```suggestion
  const options = {
    collection: "pages",
    where: {
      slug: {
        equals: slug,
      },
    },
    limit: 0,
  };

  const { docs } = await payload.find(options);
```
```